apiary.apib

```diff
@@ -607,14 +607,14 @@ Get a list of contacts.
             + (object)
                 + field (enum[string], required)
                     + Members
-                        + added_at
+                        + name - Sorts alphabetically on last_name and first_name
```

Comment:
We were not sure about the naming of this 'field'.
It's a combined field, so the options were `name` or `full name`.
Any remarks, please tell us!